packages/contracts/src/plugins/governance/multisig/MultisigSetup.sol

```diff
@@ -6,6 +6,7 @@ import {IDAO} from "../../../core/dao/IDAO.sol";
 import {DAO} from "../../../core/dao/DAO.sol";
 import {PermissionLib} from "../../../core/permission/PermissionLib.sol";
 import {PluginSetup, IPluginSetup} from "../../../framework/plugin/setup/PluginSetup.sol";
+import {IMultisig} from "./IMultisig.sol";
```

Comment:
Using it?

Reply:
Fixed 37f65aa6fa63ec515a246743564608b1e999db97